### PR TITLE
Fixed few typos

### PR DESCRIPTION
**Pull request by anisalibegic**
Here's another PR with a couple of typos fixed. As you can see there was a typo in _fileReferenceI**n**dentifiers_, in class _ParsedProjectInfo._ Maybe we should do some check on that since I'm not sure if that property is used somewhere outside Flutter?

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: examples/api/test/material/dropdown_menu/dropdown_menu.2_test.dart [hunk 1]
```diff
@@ -10,7 +10,7 @@ import 'package:flutter_api_samples/material/dropdown_menu/dropdown_menu.2.dart'
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  testWidgets('DropdownMenu cursor behavoir', (WidgetTester tester) async {
+  testWidgets('DropdownMenu cursor behaviour', (WidgetTester tester) async {
```
Comment (goderbauer) [inline examples/api/test/material/dropdown_menu/dropdown_menu.2_test.dart:13]: Can you change this to "behavior" (US english, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#spell-words-in-identifiers-and-comments-correctly)

File: packages/flutter_tools/test/integration.shard/swift_package_manager_test.dart [hunk 2]
```diff
@@ -54,12 +54,12 @@ void main() {
             expectedLines: _expectedLines(
               platform: platformName,
               appDirectoryPath: appDirectoryPath,
-              cococapodsPlugin: integrationTestPlugin,
+              cocoaPlugin: integrationTestPlugin,
```
Comment (goderbauer) [inline packages/flutter_tools/test/integration.shard/swift_package_manager_test.dart:57]: This one should be "cocoaPodsPlugin" (here and elsewhere)

**Review by goderbauer — APPROVED** [review]
LGTM after the two comments are resolved.

**Review by Renzo-Olivares — APPROVED** [review]
LGTM pending @goderbauer comments

**Comment by anisalibegic** [comment]
> LGTM after the two comments are resolved.

All done!

**Review by goderbauer — APPROVED** [review]
LGTM and thank you!

**Pull request merged**